test/e2e/common.go

```diff
@@ -20,6 +21,23 @@ func withRestartPolicy(restartPolicy corev1.RestartPolicy) podOption {
 	}
 }
 
+// Optional method to add ContainerPort and ReadinessProbe to listen Port 80
```

Comment:
I think it's not fixed value `80` but a viable value, should remove `80` in comments?